test/configCases/output/inner-dirs-entries/dep.js

```diff
@@ -0,0 +1,6 @@
+import glob from 'glob';
```

Comment:
That's just to force the common chunk

Reply:
Better use a local file (you can create a `node_modules` directory in the test case) and set `splitChunks.minSize: 0` to force a vendor chunk